core/storage/statistics/gae_models.py

```diff
@@ -64,119 +59,6 @@ def get_or_create(cls, exploration_id, state_name):
         return counter
 
 
-class StateRuleAnswerLogModel(base_models.BaseModel):
```

Comment:
I was wondering what our process for actually deleting these in the future would be...